patches/src/main/kotlin/app/revanced/patches/strava/snowplow/BlockSnowplowTrackingPatch.kt

```diff
@@ -0,0 +1,21 @@
+package app.revanced.patches.strava.snowplow
+
+import app.revanced.patcher.patch.bytecodePatch
+import app.revanced.util.returnEarly
+
+/**
+ * The app collects user interaction events and sends them to
+ * https://c.strava.com/com.snowplowanalytics.snowplow/tp2.
+ * This patch prevents the insertion of events, keeping the list of emittable events empty.
+ */
+@Suppress("unused")
+val blockSnowplowTrackingPatch = bytecodePatch(
+    name = "Block Snowplow tracking",
+    description = "See https://snowplow.io.",
```

Comment:
```suggestion
    description = "Blocks snowplow temeletry. see https://snowplow.io for more information.",
```

or analytics instead of telemetry